addons/mail/models/mail_channel_partner.py

```diff
@@ -23,6 +23,7 @@ class ChannelPartner(models.Model):
     fold_state = fields.Selection([('open', 'Open'), ('folded', 'Folded'), ('closed', 'Closed')], string='Conversation Fold State', default='open')
     is_minimized = fields.Boolean("Conversation is minimized")
     is_pinned = fields.Boolean("Is pinned on the interface", default=True)
+    last_meaningful_action_time = fields.Datetime('Last action time for the thread', default=fields.Datetime.now)
```

Comment:
As posted above, a lot of actions should not change the value of this field, only some very specific "important" actions.
So I believe we need a synonym of meaningful/important/... to be in this name to make it very clear that it's just not "any" action.

Reply:
`_time` is confusing though, so for that part I'm ok to change to `_datetime` (or `_dt` which I find a bit obscure but I guess I could get used to it).

Reply:
Well meaningful does not mean anything either ^^ . Looking at code, this is mainly linked to the channel partner being pinned again. Why not ``last_pin_dt``, ``last_interest_dt``,  ``last_alive_dt`` ?

Naming things :D . Your string also tells that it is unclear what is usage is: "last action time for the thread". Not really clear...

Reply:
The interesting cases currently include: creating/joining/pinning/new message.

Reply:
`last_interest_dt` I'm ok with if you find it more clear than meaningful

Reply:
Well it is a bit vague but at least it indicates the user did something to show its interest, which is I suppose globally ok :) . Could be great to add in help what are the actual actions updating this field (those you just highlighted), so that in the future we know when it should be updated.